modules/log6x-oc-explain.adoc

```diff
@@ -0,0 +1,17 @@
+
+// Module included in the following assemblies:
+//
+:_mod-docs-content-type: CONCEPT
+[id="log6x-oc-explain_{context}"]
+
+= Using the `oc explain` command
+
+The `oc explain` command is an essential tool in the OpenShift Command-Line Interface (CLI) that provides detailed descriptions of the fields within Kubernetes resources, including Custom Resources (CRs) introduced by Operators. This command is invaluable for administrators and developers who are configuring or troubleshooting resources in an OpenShift cluster.
```

Comment:
It should be useful an example in this section, for example, how to review the outputs and options supported for an specific output. This should be so much helpful as the most of the people doesn't know how to run it or what they need to write

`oc explain <API>.spec....`

@libander

Reply:
oc explain lokistacks.loki.grafana.com
oc explain recordingrules.loki.grafana.com
oc explain rulerconfigs.loki.grafana.com
oc explain alertingrules.loki.grafana.com
oc explain lokistacks.loki.grafana.com.spec
oc explain lokistacks.loki.grafana.com.spec.storage
oc explain lokistacks.loki.grafana.com.spec.storage.schemas.FIELDS
oc explain lokistacks.loki.grafana.com.spec.size